apps/client/src/app/import/import.component.html

```diff
@@ -34,7 +34,7 @@ <h1>Import</h1>
         <tr *ngFor="let f of files; let fi = index" [class.invalid]="(f.contentType === 0 || f.contentType === 1) && !f.resource">
           <td>
             {{f.name}}
-            <i *ngIf="f.message" class="fas fa-exclamation" style="color: red;" [ngbTooltip]="f.message"></i>
+            <i *ngIf="f.message" class="fas fa-exclamation" style="color: #ff0000;" [ngbTooltip]="f.message"></i>
```

Comment:
In future, consider making styles like this a standard CSS class. i.e.
```
i.warning {
  color: #ff0000;
}
```
Not necessary for this PR (unless you want), though.

Reply:
Did that